mailpoet/tests/integration/Segments/DynamicSegments/Filters/WooCommerceNumberOfOrdersTest.php

```diff
@@ -2,15 +2,19 @@
 
 namespace MailPoet\Segments\DynamicSegments\Filters;
 
+use MailPoet\Segments\DynamicSegments\Filters\WooCommerceNumberOfOrders;
```

Comment:
I also noticed that this line is unnecessary because the test is in the same namespace.